app/javascript/index_books.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div id="book">
+    <ul v-for="book in books">
+      <h5> {{book.title}} </h5>
+      <p> {{book.description}} </p>
+    </ul>
+  </div>
+</template>
+
+<script>
+  export default {
+    data: function () {
+      let el = document.getElementById('books_list');
+      let json = JSON.parse(el.getAttribute('data'));
```

Comment:
I would recommend to use something like [axios](https://github.com/axios/axios) to fetch data over api. It's never a good idea to fetch data from data attributes in vue

Reply:
Do you want my death?

Reply:
Let's force @isabloCh to do it in the next pull-request

Reply:
@resivalex alright 👍